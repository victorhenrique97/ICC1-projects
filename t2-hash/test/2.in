5
10
18
63
94
73
39
37
58
90
53
55
