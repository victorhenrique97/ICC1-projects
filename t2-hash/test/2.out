0: 90 55
1:
2: 37
3: 18 63 73 58 53
4: 94 39
